test/test.compute.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013 Google Inc. All Rights Reserved.
+ * Copyright 2014 Google Inc. All Rights Reserved.
```

Comment:
But if the file was originally authored in 2013 doesn't that make this change incorrect?